lib/cylc/task_pool.py

```diff
@@ -738,18 +807,17 @@ def has_unkillable_tasks_only(self):
                     return False
         return True
 
-    def poll_task_jobs(self, ids=None):
+    def poll_task_jobs(self, items=None, compat=None):
         """Poll jobs of active tasks.
 
-        If ids is specified, poll active tasks matching given IDs.
+        If items is specified, poll active tasks matching given IDs.
```

Comment:
Both are correct in their own ways. The interface specifies a list of items for matching task IDs.